pkgs/development/python-modules/pillow/default.nix

```diff
@@ -12,17 +12,21 @@
 
 import ./generic.nix (rec {
   pname = "pillow";
-  version = "9.4.0";
+  version = "9.5.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     pname = "Pillow";
     inherit version;
-    hash = "sha256-ocLXeARI65P7zDeJvzkWqlcg2ULjeUX0BWaAMX8c0j4=";
+    hash = "sha256-v1SEedM2cm16Ds6252fhefveN4M65CeUYCYxoHDWMPE=";
   };
 
+  # Reverts 04cf5e2cfc5dc1676efd9f5c8d605ddfccb0f9ee, which prevents pillow from
```

Comment:
We can't use fetchpatch with the revert argument?

Reply:
maybe, I didn't know fetchpatch had a revert argument!